invokeai/frontend/web/public/locales/en.json

```diff
@@ -327,6 +327,7 @@
         "dropOrUpload": "$t(gallery.drop) or Upload",
         "dropToUpload": "$t(gallery.drop) to Upload",
         "deleteImage": "Delete Image",
+        "deleteImage_plural": "Delete {{count}} Images",
```

Comment:
Sorry, I didn't catch this on the first review. There's a cleaner way to handle pluralization in i18next. You can use `_one` and `_other` and always provide `{ count }`:

```jsonc
// en.json
"deleteImage_one": "Delete Image",
"deleteImage_other": "Delete {{count}} Images",
```

```ts
// DeleteImageButton.tsx
t('gallery.deleteImage', { count: imageSelectionLength })
```

i18next will choose the appropriate string based on `count`. No need to have a conditional for the translation.

Reply:
I looked at the docs, tried to use the `_plural` variation, and assumed a version mismatch when it didn't work. I will adjust accordingly.